src/bump/type.go

```diff
@@ -72,3 +81,19 @@ func Bump(version *semver.Version, bt Type) *semver.Version {
 
 	return &v
 }
+
+// NameToType returns the bump type from a string. The string should be from a constant constant of bump.Name
+// or it will return bump.None.
+func NameToType(name string) Type {
+	//nolint:exhaustive // case NoneName is captured in the last default clause.
```

Comment:
If I call `NameToType("foobar")` right now I'd get `bump.None`, but that's likely not what I want, as I should never been allowed to pass "foobar" to `NameToType` and get away with it. By returning `None`, this function is hiding my mistake. Do you think it would make sense to return an error if the type is not translatable?

Reply:
If this was for internal application logic as it was until now, I would discuss it... But as it is a parameter that is provided by the user, changing the value silently is a really bad UX. You are completely right.

The change you suggest is solved in [#1453c4a](https://github.com/newrelic/release-toolkit/pull/128/commits/1453c4a6d6cb15f2de7dfa9a2f0b97ef0391055b)